components/opponent/commons/opponent.lua

```diff
@@ -251,12 +252,12 @@ function Opponent.resolve(opponent, date, options)
 		for _, player in ipairs(opponent.players) do
 			if options.syncPlayer then
 				PlayerExt.syncPlayer(player, {savePageVar = not Opponent.playerIsTbd(player)})
+			if not player.team then
+				player.team = PlayerExt.syncTeam(player.pageName:gsub(' ', '_'), nil)
+			end
 			else
 				PlayerExt.populatePageName(player)
 			end
-			if player.team then
-				player.team = TeamTemplate.resolve(player.team, date)
-			end
```

Comment:
why is this removed?

Reply:
added, I think I overdeleting these when I made the 4th update